test/Test/IO.hs

```diff
@@ -170,36 +159,26 @@ sign_ sk bs = case decodeHex (T.drop 2 bs) of
   Just dbs -> TSignature . SignatureEd25519 $ Ed25519.sign sk dbs
   Nothing -> error "Error with making signatures"
 
--- Test that no operations are called if the --dry-run flag
--- is provided in cmdline.
-test_dryRunFlag :: TestTree
-test_dryRunFlag = testGroup "Dry run does not execute any action"
-  [ testCase "Handle values correctly with placeholders" $ do
-      runMock (defaultHandlers $ defaultMockInput { miCmdLine = ["burn", "--value", "100", "--dry-run"] }) $ do
-        addExpectation ParseCmdLine Once
-        mainProgram
-  ]
-
 ---- Test Creation of multisig package. Checks the following.
----- The command is parsed correctly
 ---- Checks the package is created with the provided parameter
 ---- The replay attack counter is correct
 ---- The multisig address is correct
 ---- The expected calls are made.
 multiSigCreationTestHandlers :: Handlers TestM
 multiSigCreationTestHandlers =
-  (defaultHandlers (defaultMockInput { miCmdLine =  args}))
+  defaultHandlers
     { hReadFile = \_ -> throwM $ TestError "Unexpected file read"
-    , hRunTransactions = \_ _ _ _ -> throwM $ TestError "Unexpected `runTransactions` call"
-    , hGetTezosClientConfig = pure $ Right ("tezos-client", cc)
-    , hGetAddressForContract = \ca ->
-        if ca == "tzbtc"
-          then pure $ Right contractAddress
-          else if ca == "tzbtc-multisig"
-            then pure $ Right multiSigAddress
-            else throwM $ TestError $ "Unexpected contract alias: " ++ (toString ca)
-    , hGetStorage = \x -> if x == multiSigAddressRaw
-      then pure $ nicePackedValueToExpression (mkStorage 14 3 [])
+    , hInjectOperation = \_ -> throwM $ TestError "Unexpected `injectOperation` call"
+    , hGetTezosClientConfig = throwM $ TestError "Unexpected tezos-client get config call"
+    , hResolveAddressMaybe = \case
+        AddressResolved addr -> pure $ Just addr
+        AddressAlias (Alias a) -> case a of
+          "tzbtc" -> pure $ Just contractAddress
+          "tzbtc-multisig" -> pure $ Just multiSigAddress
+          _ -> pure $ Nothing
+    , hGetContractScript = \x -> if x == multiSigAddress
+      then pure $ OriginationScript
+           (toExpression . toVal $ mkStorage 14 3 []) (toExpression . toVal $ mkStorage 14 3 [])
```

Comment:
Is it safe to put a value where code should be?

Reply:
`hGetContractScript` is used only to get contract storage in that scenario, so I assume that it's more or less safe to put some nonsense to the code, however, it should at least be commented